ci/code-quality/whitespace-lints.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -euo pipefail
 
+# Use ggrep if available (for macOS), otherwise grep.
+GREP=$(command -v ggrep || command -v grep)
```

Comment:
since I am using `type` later - can we use it instead of `command -v` to stay consistent ?